Gruntfile.js

```diff
@@ -228,6 +228,11 @@ module.exports = (grunt) => {
         src: ['**/*', '!style/**', '!svgs/**'],
         dest: distAssets,
         expand: true
+      },
+      strings: {
+        src: ['strings/*.json', '!strings/english.json'],
+        dest: distAssets,
+        expand: true
```

Comment:
Why is `english.json` being excluded here? I see no reason not to copy it just in case.

Reply:
Since `english.json` is the default file, it kinda means "no translation at all": all its values are identical to its keys. Therefore correct translation code should not need it.

Reply:
It's true that at the moment `english.json` isn't being used. However, that doesn't necessarily mean that it won't ever be used at any point in the future. When we write code with exceptions like this, it creates traps / time-bombs for the future if the code changes at any point in time to need it. In this case, such an exception isn't needed. It doesn't save us anything, and instead creates cognitive overhead by creating an exceptional circumstance. The app won't be slower if we include this file, nor will it break anything, and it will result in less logic if you remove the exception, so I think it's a good idea to copy over that file and leave the logic simple.

Reply:
The main intent was to avoid copying one more .json file on every build, but I agree it is no big deal

Reply:
I also want to make sure that file won't be downloaded by the browser unless we actively use it

Reply:
> I also want to make sure that file won't be downloaded by the browser unless we actively use it

OK, that part makes sense. It's still OK to copy the file though so that it's available if we decide we want it for some reason.